### ISSUE TITLE
[question] rpm-adapter does not generate sqlite DBs,  has any  plan to implement? 

### ISSUE THREAD

**Issue by lugeng** [artipie/rpm-adapter#514]

**Comment by g4s8** [comment]
@lugeng good point! Sqlite operations should be much faster than using xml files, we definitely need to support it.

**Comment by olenagerasimova** [comment]
DB structure is described [here](https://github.com/rpm-software-management/createrepo_c/blob/master/src/sqlite.c)

**Comment by Conan-Kudo** [comment]
The SQLite export is deprecated in createrepo, and isn't used by any of the currently maintained RPM package managers. I wouldn't bother implementing it at all.
